server/config/config.go

```diff
@@ -0,0 +1,96 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+var cfgs *Configs
+
+type Configs struct {
```

Comment:
i like the `Config` structure, maybe we should move all the options to here instead of using flag